src/Utilities/CommonUtils.tsx

```diff
@@ -1,38 +1,46 @@
 import Home from "../PortfolioContainer/Home/Home";
 import AboutMe from "../PortfolioContainer/AboutMe/AboutMe";
 import Resume from "../PortfolioContainer/Resume/Resume";
+import NewResume from "../PortfolioContainer/Resume/NewResume";
 
 export type ScreenProps = {
-  screen_name: string
-}
+  screen_name: string;
+};
 
 export type Screen = {
-  component: (props: ScreenProps) => JSX.Element,
-  alreadyRendered: boolean
-  props: ScreenProps
-}
-
+  component: (props: ScreenProps) => JSX.Element;
+  alreadyRendered: boolean;
+  props: ScreenProps;
+};
 
 export const TOTAL_SCREENS = new Array<Screen>(
   {
     component: Home,
     alreadyRendered: false,
     props: {
-      screen_name: "Home"
-    }
-  }, {
-  component: AboutMe,
-  alreadyRendered: false,
-  props: {
-    screen_name: "About_Me"
-  }
-},
+      screen_name: "Home",
+    },
+  },
+  {
+    component: AboutMe,
+    alreadyRendered: false,
+    props: {
+      screen_name: "About_Me",
+    },
+  },
+  // {
+  //   component: Resume,
+  //   alreadyRendered: false,
+  //   props: {
+  //     screen_name: "Resume",
+  //   },
+  // },
   {
-    component: Resume,
+    component: NewResume,

```

Comment:
Instead of calling new Components "New" it's often better to call the old Components "Old". That way you don't find to find and update references such as this.